src/components/Documentation/RightPanel/index.tsx

```diff
@@ -78,7 +107,10 @@ const RightPanel: React.FC<IRightPanelProps> = ({
   }
 
   useEffect(() => {
-    const throttledSetCurrentHeader = throttle(updateCurrentHeader, 100)
+    const throttledSetCurrentHeader = (): void => {
+      throttle(updateCurrentHeader, 100)()
+      debounce(scrollSidebar, 100)()
```

Comment:
The extra `()` in `throttle(updateCurrentHeader, 100)()` looks weird, but that's because lodash throttle returns a function for invocation. I don't use lodash much, so at first this confused me, but thanks to VSCode tooling ❤️ I can easily see what going on under the hood.

![image](https://user-images.githubusercontent.com/22982964/124348093-93b05780-dc1a-11eb-9c23-d2fe2000248a.png)